src/main/java/com/example/services/implementations/NotesServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.example.services.implementations;
+
+import com.example.dao.NotesDao;
+import com.example.entity.Notes;
+import com.example.services.NotesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service("notesService")
+public class NotesServiceImpl implements NotesService {
+
+    @Autowired
+    protected NotesDao notesDao;
+
+    @Override
+    public Notes save(Notes notes){
```

Comment:
final and anywhere

Reply:
@Vlada0612 I'm afraid the final parameter is not very suitable for me here.

Reply:
don't afraid